File: 2-semestre/uri-online-judge/javascript/1156.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let s = 1;
let aux = 2;

for (let i = 3; i <= 39; i += 2) {
    s += i / aux;
    aux *= 2;
}

console.log(`${s.toFixed(2)}`);

File: 2-semestre/uri-online-judge/c/1156.c
#include <stdio.h>

int main() {
    double s = 1, i, aux = 2;

    for (i = 3; i <= 39; i+=2) {
        s += i / aux;
        aux *= 2;
    }

    printf("%.2f\n", s);

    return 0;
}
